presto-main/src/main/java/com/facebook/presto/sql/planner/plan/Patterns.java

```diff
@@ -227,6 +227,11 @@ public static Property<PlanNode, PlanNode> source()
                 Optional.empty());
     }
 
+    public static Property<SemiJoinNode, PlanNode> filteringSource()
```

Comment:
Put it in a statics class "SemiJoin" like other node specific properties below

Reply:
fixed